spec/views/shared/activities/activity_spec.rb

```diff
@@ -15,11 +15,11 @@
       allow(view).to receive(:edit_activity_redaction_path).and_return("This path isn't important")
       allow(view).to receive(:organisation_activity_path).and_return("This path isn't important")
     end
-
-    render
```

Comment:
Is there an example that this is being removed from? Looks like it shows up in every context that follows?

Reply:
The annoying thing about `render`ing at the top is that it's already rendered by the time we try to do other `before`s in individual contexts, therefore the tests were falsely failing. So I've had to move it as lower down as needed for specific scenarios.

There may be better ways to do the entire thing (i.e. testing for the presence or absence of content and links), but it would be a bigger time and energy commitment to make sure the important cases are still covered.